packages/admin-ui/src/card/card-title.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import type { ComponentPropsWithRef, ReactNode } from 'react'
+import { createComponent, useElement } from '@vtex/admin-ui-react'
+
+import { Paragraph } from '../components/Paragraph'
+import { Stack } from '../stack'
+
+export const CardTitle = createComponent<'div', CardTitleOptions>((props) => {
+  const { children, label, thumbnail, ...restProps } = props
+
+  return useElement('div', {
+    ...restProps,
+    children: (
+      <Stack direction="row" space="$l">
+        {thumbnail ? (
+          <CardThumbnail src={thumbnail.src} alt={thumbnail.alt} />
+        ) : null}
+        <Paragraph
+          className="__admin-ui-card-nested-title"
+          csx={{ text: '$title1' }}
```

Comment:
Are you considering this point? It seems they will have the same token 🤔
![Screen Shot 2022-07-20 at 10 54 47](https://user-images.githubusercontent.com/991309/180000783-d5a5b85b-0efb-40b8-8d09-b2b45ba84561.png)